ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -679,7 +681,13 @@ static void HandleKernelRequest(void* messageMemory, uint32_t messageSize)
     // whereas messages originating in the kext's vnode handler will only fill
     // the fsid/inode, so we need to look up the path below.
```

Comment:
This comment is now a bit outdated:

```
// We expect a non-null request.path for messages sent from the FILEOP handler,
// whereas messages originating in the kext's vnode handler will only fill
// the fsid/inode, so we need to look up the path below.
```

Could you update to include all of the possible scenarios?

Reply:
Since we simplied path to send "" when non-relavent, this comment is valid again.